lib/models/user-whitelist.js

```diff
@@ -13,4 +14,52 @@ util.inherits(UserWhitelist, Base);
 
 require('../extend-with-factories')(UserWhitelist);
 
+/**
+ * Is an organization in a trial period?
+ * @returns {boolean}
+ */
+UserWhitelist.prototype.isInTrial = function () {
+  return moment(this.attrs.trialEnd) > moment.utc();
```

Comment:
I think it's nicer to use the `isBefore` and `isAfter syntax in moment, since it makes it a bit more clear and you don't have to deal with UTC or timestamps.